### PR TITLE
Clean up coding style

### PR DESCRIPTION
**Pull request by MRZA-MRZA**
According to https://github.com/KaidanIM/Kaidan/issues/23 coding style is fixed.

### CODE REVIEW

File: src/qml/LoginPage.qml [hunk 1]
```diff
@@ -2,37 +2,38 @@ import QtQuick 2.5
...
+		Label {
 			id: statusLabel
 			text: "Not connected"
 		}
 	}
-	Component.onCompleted:{
-		function goToRoster(){
+	Component.onCompleted: {
+		function goToRoster()
+		{
```
Comment (LNJ2) [inline src/qml/LoginPage.qml]: I have to say in JS, I like this style more (as in #23 )
```javascript
function goToRoster() {
```

And please add a blank (or a new line) for the function in line 41/42.

**Comment by LNJ2** [comment]
And please use the present tense for commit descriptions (Fix xyz, instead of Fixed xyz).
And you can add "closes #23" or "Closes #23." to the commit description (please in a new line), then github will close that issue when the PR will be merged.

**Comment by LNJ2** [comment]
Please correct the style of the function in src/qml/LoginPage.qml:41

File: src/qml/LoginPage.qml [hunk 2]
```diff
@@ -2,43 +2,46 @@ import QtQuick 2.5
...
+	Component.onCompleted: {
+		function goToRoster() {
 			statusLabel.text = "Connected";
 			//we need to disconnect enableConnectButton to prevent calling it on normal disconnection
 			kaidan.connectionStateDisconnected.disconnect(enableConnectButton)
 			mainLoader.source = "RosterPage.qml"
 		}
-		function enableConnectButton(){connectButton.enabled = true}
+		function enableConnectButton()
+		{
```
Comment (LNJ2) [inline src/qml/LoginPage.qml]: Please as above.

Comment (MRZA-MRZA) [inline src/qml/LoginPage.qml]: Sorry, I automatically use same coding style everywhere.

**Comment by LNJ2** [comment]
@JBBgameich Thank you for speaking German! :trollface: :no_entry: :de: :no_entry_sign:

**Pull request merged**